roles/custom/matrix-bot-draupnir/defaults/main.yml

```diff
@@ -72,12 +72,12 @@ matrix_bot_draupnir_access_token: ""
 matrix_bot_draupnir_pantalaimon_username: ""
 matrix_bot_draupnir_pantalaimon_password: ""
 
-# Username and password the bot uses for logging in directly. If using Pantalimon
+# Username and password the bot uses for logging in directly. If using Pantalaimon
 # these values become the values of `matrix_bot_draupnir_pantalaimon_username` and `matrix_bot_draupnir_pantalaimon_password`
 matrix_bot_draupnir_login: "{{ matrix_bot_draupnir_pantalaimon_username if matrix_bot_draupnir_pantalaimon_use == 'true' else 'bot.draupnir' }}"
 matrix_bot_draupnir_password: "{{ matrix_bot_draupnir_pantalaimon_password }}"
 
-# Controls if we activate the pantalaimon config block for now. This configuration block is proable to change name for our usecase
```

Comment:
I interpreted "proable" as "probably", not something like "pro-able".

Reply:
its a typo of probable. The new wording doesnt change the meaning too much if at all.

As for why im not spelling Pantal with Capitalisation. Its not a name in this case its literally what the configuration block is called.

Reply:
> Its not a name in this case its literally what the configuration block is called.

This statement is not quite understandable.